utils/utils.js

```diff
@@ -652,6 +652,10 @@ const getDataFromSource = (src, dest, properties) => {
   return data;
 };
 
+const removeTrailingSlash = (str) => {
```

Comment:
Let's rename this to `removeTrailingSlashes` for improving clarity.

Reply:
yeah updating